Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[workspace]
+members = [
+  "game_engine",
+  "game_server",
+  "client/graphics",
```

Comment:
for now, I would remove line 5 and 6

Reply:
Let's see. First of all it seems as `cargo` doesn't support "nested" workspaces, also we I'm working on automating a lot of stuff using `cargo make foobar`.... If that doesn't work out as I hope or doesn't satisfy you, we can still split this up :)

Reply:
So, the cargo-make integration appears to work overall, so I'll resolve this for now. If anybody disagrees, feel free to reopen :)